resources/views/pages/Public/Join.svelte

```diff
@@ -8,98 +8,135 @@
+      </div>
+      <div class="w-full px-3 md:w-1/2">
+          <label class="mb-2 block text-xs font-bold uppercase tracking-wide text-gray-200" for="grid-age">
+              Your age
+          </label>
+          <input
+              class="block w-full appearance-none rounded border border-gray-200 bg-gray-200 py-3 px-4 leading-tight text-gray-700 focus:border-gray-500 focus:bg-white focus:outline-none invalid:border-red-500"
+              id="grid-age"
+              type="text"
+              placeholder="e.g. 27"
+              required
+          />
+      </div>
+  </div>
+  <div class="-mx-3 mb-6 flex flex-wrap">
+      <div class="mb-6 w-full px-3 md:mb-0 md:w-1/2">
+          <label class="mb-2 block text-xs font-bold uppercase tracking-wide text-gray-200" for="grid-location">
+              Location
+          </label>
+          <input
+              class="mb-3 block w-full appearance-none rounded border border-gray-200 bg-gray-200 py-3 px-4 leading-tight text-gray-700 focus:border-gray-500 focus:bg-white focus:outline-none invalid:border-red-500"
+              id="grid-location"
+              placeholder="Make it as long and as crazy as you'd like."
+              required
+          />
+          <p class="text-xs italic text-gray-400">You can be as specific as you like</p>
+      </div>
+      <div class="mb-6 w-full px-3 md:mb-0 md:w-1/2">
+          <label class="mb-2 block text-xs font-bold uppercase tracking-wide text-gray-200" for="grid-weekly">
+              Are you available for weekly operations?
+          </label>
+          <div class="relative">
+              <select
+                  class="block w-full appearance-none rounded border border-gray-200 bg-gray-200 py-3 px-4 pr-8 leading-tight text-gray-700 focus:border-gray-500 focus:bg-white focus:outline-none invalid:border-red-500"
+                  id="grid-weekly"
+                  required
+              >
+                  <option value="" disabled selected class="text-gray-200">Please Select an Option</option>
+                  <option>No</option>
+                  <option>Yes</option>
+              </select>
+          </div>
+            <p class="mt-2 text-xs italic text-gray-400">
+              Arma - Saturday at 1800 Zulu
+              DCS - Sunday at 1800 Zulu
+            </p>
+      </div>
+  </div>
+  <div class="-mx-3 mb-6 flex flex-wrap">
+    <div class="mb-6 w-full px-3 md:mb-0 md:w-1/2">
+        <label class="mb-2 block text-xs font-bold uppercase tracking-wide text-gray-200" for="grid-discord">
+            Discord Username
+        </label>
+        <input
+            class="mb-3 block w-full appearance-none rounded border border-gray-200 bg-gray-200 py-3 px-4 leading-tight text-gray-700 focus:border-gray-500 focus:bg-white focus:outline-none invalid:border-red-500"
+            id="grid-discord"
+            type="text"
+            placeholder="Your Discord Username."
```

Comment:
Some placeholders end with a `.` and some don't, to be consistent I think we should remove the `.`

Reply:
Fixed this, should all be consistent from next push